Src/WitsmlExplorer.Api/HttpHandlers/ObjectHandler.cs

```diff
@@ -0,0 +1,19 @@
+using System.Threading.Tasks;
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+using WitsmlExplorer.Api.Models;
+using WitsmlExplorer.Api.Services;
+
+namespace WitsmlExplorer.Api.HttpHandlers
+{
+    public static class ObjectHandler
+    {
+        [Produces(typeof(ObjectOnWellbore[]))]
+        public static async Task<IResult> GetObjectsIdOnly(string wellUid, string wellboreUid, EntityType objectType, IObjectService objectService)
```

Comment:
`GetObjectsIdOnly` seems ambigious. Is it possible to use a more clear name?

Reply:
It fits with the rest of the repo when talking about Object(OnWellbore). I wish we had a better name for it tho.